src/main/flight/pid.c

```diff
@@ -1050,6 +1052,19 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     const bool launchControlActive = isLaunchControlActive();
 
+    const bool levelModeActive = FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE) || FLIGHT_MODE(GPS_RESCUE_MODE);
+    static timeUs_t levelModeStartTimeUs = 0;
```

Comment:
Coding standard in Betaflight is to put the declaration of function scope static variables at the beginning of the function defining them.

Reply:
Since the stylistic fix was so small I just amended the commit and force-pushed.